src/test/lib/HandshakeTest.cpp

```diff
@@ -2075,3 +2075,101 @@ QuicTestConnectClientCertificate(
         }
     }
 }
+
+void
+QuicTestInvalidAlpnLengths(
+    void
+    )
+{
+    int Lengths[] = { 0, QUIC_MAX_ALPN_LENGTH + 1 };
+    char AlpnBuffer[QUIC_MAX_ALPN_LENGTH + 3]; // + 3 so it can always be 0 terminated
+    for (int Len = 0; Len < (int)ARRAYSIZE(Lengths); Len++)
+    {
```

Comment:
nit: formatting (below as well)
```suggestion
    for (int Len = 0; Len < (int)ARRAYSIZE(Lengths); Len++) {
```